tests/python_package_test/test_engine.py

```diff
@@ -1803,6 +1803,51 @@ def train_booster(params=params_obj_verbose, **kwargs):
         self.assertRaises(lgb.basic.LightGBMError, get_cv_result,
                           params_class_3_verbose, metrics='binary_error', fobj=dummy_obj)
 
+    def test_multiple_feval_train(self):
+        X, y = load_breast_cancer(True)
```

Comment:
It will cause errors in future versions of scikit-learn.
```suggestion
        X, y = load_breast_cancer(return_X_y=True)
```